src/main/java/de/refactoringbot/refactoring/supportedrefactorings/RenameMethod.java

```diff
@@ -129,12 +129,13 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 
 		// If refactor-method not found
 		if (methodToRefactor == null) {
-			throw new BotRefactoringException("Could not find specified method! Automated refactoring failed.");
+			throw new BotRefactoringException(
+					"Could not find specified method declaration at given line! Please check if you placed your comment on a method declaration.");
```

Comment:
Same issue with the second part of the message as already mentioned in a previous comment. Of course, here the special case is that this refactoring can be triggered so far only via comments.